.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
```

Comment:
hmm, many look legit to me, but you've done a lot here already so why don't we merge these for now. i think the intent of this is more around getting this in place, which you've achieved:
Run pep8 and ESLint on commit? 
i'll create a separate issue for housecleaning those and later removing the E501.